core/src/main/java/com/rafaskoberg/gdx/parrot/ParrotSettings.java

```diff
@@ -16,14 +17,24 @@ public class ParrotSettings {
     public int loudnessExponentialCurve = 3;
 
     /**
-     * In the spatial system, distances beyond this limit will have the maximum amount of reduction. Defaults to 15
+     * In the spatial system, distances beyond this limit will be based off {@link float distanceRolloffMinGain}. Defaults to 15
      */
-    public float soundDistanceLimit = 15f;
+    public float distanceRolloffMaxDistance = 15f;
 
     /**
-     * Amount of reduction applied to sounds, in decibels, according to their distance. Defaults to -2
+     * In the spatial system, distances before this limit will be played with no distance volume dampening. Distances beyond this limit will drop off based off {@link Interpolation distanceRolloffCurve}. Defaults to 3
      */
-    public float soundDistanceReduction = -2f;
+    public float distanceRolloffMinDistance = 3f;
+
+    /**
+     * In the spatial system, volume dampening based off distances use this to determine the volume multiplier beyond {@link float distanceRolloffMaxDistance}. Defaults to 0
+     */
+    public float distanceRolloffMinGain = 0f;
+
+    /**
+     * In the spatial system, distances beyond {@link float distanceRolloffMinDistance} will be based off this drop off curve and {@link float distanceRolloffMinGain}. Defaults to exp5In
+     */
+    public Interpolation distanceRolloffCurve = Interpolation.exp5In;

```

Comment:
I think we should use `exp3in` as the default interpolation function because it's almost identical to `exp5in` and matches the power `3` used in `loudnessExponentialCurve`, so if users decide to change that to something crazy like 7, they can just create a `pow(7)` Interpolation function as well.

```suggestion
    /**
     * In the spatial system, distances beyond {@link #distanceRolloffMinDistance} will be based off this drop off curve and {@link #distanceRolloffMinGain}. Defaults to exp5In
     */
    public Interpolation distanceRolloffCurve = Interpolation.exp3In;

```